### ISSUE TITLE
Wrong values of % target

### ISSUE THREAD

**Issue by sebastien-foulle** [fbdesignpro/sweetviz#116]
Hello,

the html report produced by the following script shows that if bill_length_mm <= 35 then % target <  90%,
and if 35 <= bill_length_mm <= 37.5 then % target > 105% (!).

![image](https://user-images.githubusercontent.com/16180557/164770538-0abad112-d47b-4b64-8c69-e794f5fb9a30.png)

```
import pandas as pd
from palmerpenguins import load_penguins
import sweetviz as sv
penguins = load_penguins()
penguins["target"] = penguins.species == 'Adelie'
penguins = penguins[["species", "bill_length_mm", "target"]]
penguins.head()

my_report = sv.analyze(penguins, target_feat = "target")
my_report.show_html()
```

But in fact if  bill_length_mm <= 40, % target should always be 100% : there are only Adelie penguins in this case.
```
# Adelie    100
penguins.query('bill_length_mm <= 40').species.value_counts()
```
Maybe it's a rounding problem.

**Comment by fbdesignpro** [comment]
@sebastien-foulle thank you for reporting this, I will take a look!

**Comment by makotu1208** [comment]
I am experiencing a same event.
How is the progress of the investigation and fix here?

**Comment by cwzkevin** [comment]
I have a similar issue! Attached is the example_data.pkl file, [example_data.pkl.zip](https://github.com/fbdesignpro/sweetviz/files/10187589/example_data.pkl.zip)

The code to reproduce the result:
```
feature_config = sv.FeatureConfig(force_cat=['numerical_var'])
correct_report = sv.analyze([example_data, 'Train'],
                             target_feat='outcome', 
                             feat_cfg=feature_config,
                             pairwise_analysis='off')
correct_report.show_html('correct_report.html')

feature_config = sv.FeatureConfig(force_num=['numerical_var'])
wrong_report = sv.analyze([example_data, 'Train'],
                           target_feat='outcome', 
                           feat_cfg=feature_config,
                           pairwise_analysis='off')
wrong_report.show_html('wrong_report.html')
```

When we force_cat the numerical_var, we can get the correct distribution of the outcome:

![correct_need_to_force_cat](https://user-images.githubusercontent.com/21072230/206513689-e170886d-de43-493d-b8e4-b3af2ba10cbd.PNG)

If we force_num the numerical_var, the outcome distribution is completely off:

![wrong_as_numerical](https://user-images.githubusercontent.com/21072230/206513712-05652ccb-2f67-4c9e-8686-580f023fa41d.PNG)

**Comment by fbdesignpro** [comment]
Fixed by 2ec0848!
